src/Toml/Printer.hs

```diff
@@ -171,13 +179,17 @@ prettyTableArrays options i pref = mapOrdered arrText options
 
 -- Returns an indentation prefix
 tabWith :: PrintOptions -> Int -> Text
-tabWith options n = Text.replicate (n * indent options) " "
+tabWith PrintOptions{..} n = Text.replicate (n * printOptionsIndent) " "
 
 -- Returns a proper sorting function
-mapOrdered :: Ord k => ((k, v) -> [t]) -> PrintOptions -> HashMap k v -> [t]
-mapOrdered f options
-    | shouldSort options = concatMap f . sortOn fst . HashMap.toList
-    | otherwise          = concatMap f . HashMap.toList
+mapOrdered :: ((Key, v) -> [t]) -> PrintOptions -> HashMap Key v -> [t]
```

Comment:
`mapOrdered` function should take `[(Key, v)]` instead of `HashMap`. This will make the function more performant. In the implementation of this function you're calling `toList` function anyways. But when you convert `HashMap Piece v` to `HashMap Key v` you spend extra time converting back to `HashMap` which is redundant, since you are going to work with the list anywas, so you can convert `HashMap Piece v` to `[(Key, v)]`

Reply:
Got it. Sorry, this code wasn't ready for re-review :) I will definitely do this!